<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/context_actions/res/settings-small.png</normaloff>:/new/context_actions/res/settings-small.png</iconset>
  </property>
  <widget class="QTabWidget" name="twTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Sources</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>All cameras:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwSources">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbAddCamera">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add </string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera details:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTitle">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MJPEG stream URL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSourceUrl">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDeleteCamera">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSaveCamera">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Audio source URL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAudioUrl">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Authentication</string>
    </attribute>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>User name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leLogin">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePassword">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;If you have guest access enabled, leave fields above blank&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Recording</string>
    </attribute>
    <widget class="QGroupBox" name="gbRecOptions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>401</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Record after motion ceases (seconds):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leDirectory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>381</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Store to following directory:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbPostMotion">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbDefaultQuality">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>371</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Record directly from source (faster)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>MJPEG frame quality:</string>
      </property>
     </widget>
     <widget class="QSlider" name="sldRecordQuality">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>70</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Keep recordings upto (days):</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbKeepPeriod">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>59</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbRecMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Mode</string>
     </property>
     <widget class="QRadioButton" name="rbNoRecord">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>No recording</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbRecOnMotion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Record on motion detected</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Miscellaneous</string>
    </attribute>
    <widget class="QGroupBox" name="gbView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Grid</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of columns:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbShowLevel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Show motion level bars</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbColumns">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>59</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbRepair">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>391</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Camera remote reboot/other action</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>External command line:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leExtCommand">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>381</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;You can use %host as substitution for source hostname/IP address&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;in command line&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>twTabs</tabstop>
  <tabstop>lwSources</tabstop>
  <tabstop>pbDeleteCamera</tabstop>
  <tabstop>pbAddCamera</tabstop>
  <tabstop>leTitle</tabstop>
  <tabstop>leSourceUrl</tabstop>
  <tabstop>leAudioUrl</tabstop>
  <tabstop>pbSaveCamera</tabstop>
  <tabstop>rbNoRecord</tabstop>
  <tabstop>rbRecOnMotion</tabstop>
  <tabstop>leDirectory</tabstop>
  <tabstop>sbKeepPeriod</tabstop>
  <tabstop>sldRecordQuality</tabstop>
  <tabstop>cbDefaultQuality</tabstop>
  <tabstop>sbPostMotion</tabstop>
  <tabstop>sbColumns</tabstop>
  <tabstop>cbShowLevel</tabstop>
  <tabstop>leExtCommand</tabstop>
  <tabstop>pbOK</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>leLogin</tabstop>
  <tabstop>lePassword</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbDefaultQuality</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>toggleQualitySlider(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>toggleSaveButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSourceUrl</sender>
   <signal>textEdited(QString)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>toggleSaveButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SettingsDialog</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddCamera</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>toggleAddingMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDeleteCamera</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>deleteSelectedItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwSources</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>showCameraItemDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSaveCamera</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>saveCameraItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNoRecord</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>showRecordModeDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbRecOnMotion</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>showRecordModeDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleQualitySlider(bool)</slot>
  <slot>toggleSaveButton()</slot>
  <slot>saveSettings()</slot>
  <slot>showCameraItemDetails()</slot>
  <slot>toggleAddingMode()</slot>
  <slot>deleteSelectedItem()</slot>
  <slot>saveCameraItem()</slot>
  <slot>showRecordModeDetails()</slot>
 </slots>
</ui>
